js/main.js

```diff
@@ -1,27 +1,196 @@
-let myLibrary = [];
+const myLibrary = [];
+const bookCollectionContainer = document.querySelector('.book-collection-container');
+const modalTitle = document.querySelector('.modal-title');
+const feedbackMessage = document.createElement('div');
+const modalHeader = document.querySelector('.modal-header');
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+const pagesInput = document.querySelector('#pages');
+const trueRadioBtn = document.querySelector("input[id='true']");
+const falseRadioBtn = document.querySelector("input[id='false']");
+const addBookButton = document.querySelector('.add-book-btn');
+/* const modalContainer = document.querySelector('#exampleModal');
+const myModal = new bootstrap.Modal(modalContainer); */
+const toggleModalLink = document.querySelector('#toggleModalLink');
+feedbackMessage.textContent = 'Please check all fields have been completed.';
+feedbackMessage.style.width = '100%';
+feedbackMessage.classList.add('text-center', 'bg-danger');
 
+function bookStatusText(status) {
+  return `${status ? 'Already Read' : 'Not Read'}`;
+}
+
+function toggleBtnStyle(btn) {
+  btn.classList.toggle('active');
+}
+
```

Comment:
- [ ] OPTIONAL - This should be required but has been marked as optional because it wasn't requested in the previous review: Consider changing this and all other regular functions to es6 arrow functions.